### PR TITLE
Quora module added

### PR DESCRIPTION
**Pull request by Pk13055**
* [x] Updated `__init__.py ` to include the Quora class.
* [x] Added the _button_ element to the template for quora selection.
* [x] Updated feedgen accordingly.

### CODE REVIEW

**Comment by Pk13055** [comment]
@mariobehling Please review.

**Comment by mariobehling** [comment]
Please resolve conflicts. Cheers.

**Comment by Pk13055** [comment]
I have resolved the conflicts. Please review and merge accordingly. @mariobehling

File: app/scrapers/quora.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                is_proper = True
+            except:
+                is_proper = False
+            
+            for ans, body in zip(ans_temp, ans_bodies):
+                answer = {}
+                answer['author'] = {
+                    'name' : ans.findAll('a', {'class' : 'user'})[0].getText(),
+                    'description' : ans.findAll('span', { 'class' : \
+                        'IdentityNameCredential NameCredential'})[0].getText()[1:].strip(' ')
```
Comment (AnshulMalik) [inline app/scrapers/quora.py]: I see, you are using `findAll` and then extracting the first child element by `[0]`, using `find` here would be more appropriate, which only returns the first matching child.

https://www.crummy.com/software/BeautifulSoup/bs3/documentation.html#findall%20example%20document

File: app/scrapers/quora.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            
+            ans_temp = soup.findAll('div', {'class' : "Answer AnswerBase"})
+            ans_bodies = soup.select("div.ExpandedAnswer")
+            
+            # make sure answers are parsed properly
+            try:
+                assert(len(ans_temp) == ans_bodies)
+                is_proper = True
+            except:
+                is_proper = False
```
Comment (AnshulMalik) [inline app/scrapers/quora.py]: Where is `is_proper` is being used?

File: app/scrapers/quora.py (hunk 1 above)
Comment (Pk13055) [reply app/scrapers/quora.py]: Check [this](https://stackoverflow.com/questions/9819141/python-beautifulsoup-different-results-with-find-and-findall) for clarification. I just did it as a safety measure.

Comment (AnshulMalik) [reply app/scrapers/quora.py]: That issue is different, I can't find it relevant here

**Comment by Pk13055** [comment]
@nikhilrayaprolu Please review.

File: app/scrapers/quora.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    """Scrapper class for quora"""
+    def __init__(self):
+        pass
+
+    def get_page(self,query):
+        """ Fetch the quora search results page
+        Returns : Results Page
+        """
+        query_string = '-'.join(query.strip(' ').split(' '))
+        response = requests.get('https://www.quora.com/%s' %(query_string))
```
Comment (AnshulMalik) [inline app/scrapers/quora.py]: I was just looking at quora and I found https://www.quora.com/search?q=<query> as the search link?

**Comment by AnshulMalik** [comment]
I see this when I search, should it look like this  @Pk13055 ? 
aaa

**Comment by Pk13055** [comment]
No it should not. I shall check and get back to you. Last I checked however, it was working fine, are you sure you are on my branch? 

_Sent from my OnePlus ONE A2003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: app/scrapers/quora.py (hunk 3 above)
Comment (Pk13055) [reply app/scrapers/quora.py]: That was for questions containing terms, when you go to a specific question with multiple answers, the query is what I have used. I assumed that people using the service for quora would be using it to display answers for the question/query and not question(s) for the query.

**Comment by mariobehling** [comment]
Any update on this PR ;-)

**Comment by mariobehling** [comment]
Closing this due to inactivity. Please reopen once you are available again. Cheers.

**Pull request closed without merging**